collectors/ps_constants/core.py

```diff
@@ -34,9 +34,13 @@ def sync_compliance_priority(source_dict):
     sync compliance priority data
     """
     CompliancePriority.objects.all().delete()
-    for ps_module, ps_component_list in source_dict.items():
-        for ps_component in ps_component_list:
-            CompliancePriority(ps_module=ps_module, ps_component=ps_component).save()
+    for ps_module, json_field in source_dict.items():
+        components = json_field.get("components", [])
+        # Streams must not be missing, crash if missing.
+        streams = json_field["streams"]
```

Comment:
The approach is basically correct but I would be for some more readable error message at least including the name of the module which has the empty list of streams.

Reply:
Fixed.